src/utils.ts

```diff
@@ -100,8 +99,32 @@ function readCompilerOptions(configPath: string) {
   return parsedConfig.options;
 }
 
+export function getTSConfigOptionFromConfig(globals: any) {
+  if (!globals) return 'tsconfig.json';
+  
+  const tsJestConfig = getTSJestConfig(globals);
+
+  if (('__TS_CONFIG__' in globals) && globals.__TS_CONFIG__) {
+    console.warn('Using globals > __TS_CONFIG__ option for setting TS config is deprecated. Please set config using this option: globals > ts-jest > tsConfig (object) or tsConfigFile (string). More information at https://github.com/kulshekhar/ts-jest#tsconfig');
```

Comment:
Add a newline character before `globals > ts-jest > tsConfigFile (string)`. That'll make this easy to read